aiobotocore/_endpoint_helpers.py

```diff
@@ -14,6 +19,15 @@
     asyncio.TimeoutError,
 ]
 
+if httpx is not None:
+    # TODO: Wild guesses after looking at https://pydoc.dev/httpx/latest/classIndex.html
+    # somebody with more network and/or httpx knowledge should revise this list.
+    _aiohttp_retryable_exceptions.extend(
+        (httpx.NetworkError, httpx.ConnectTimeout)
+    )
+
+
+# TODO [httpx]: determine retryable exceptions
```

Comment:
should add these to TODO in PR description

Reply:
oops, had double TODO's in the code. I have a first pass at retryable exceptions, but should probably be revised. Rewritten to be less confusing & clarified in the PR description